src/client/components/workspace/TestContainer.tsx

```diff
@@ -51,19 +51,35 @@ const TestContainer: React.FC<TestContainerProps> = ({
   const [runScheduledTests, setScheduledTests] = useState<boolean>(false);
   const [callsPerSecond, setCallsPerSecond] = useState<number>(1);
   const [totalTime, setTotalTime] = useState<number>(10);
-  const [abortController, setAbortController] = useState<AbortController | null>(null);
+  const [abortController, setAbortController] =
+    useState<AbortController | null>(null);
   const isDark = useSelector((state: any) => state.ui.isDark);
 
   const handleShowLoadTest = () => {
     setShowLoadTest(!showLoadTest);
   };
 
-
   const [showLoadTest, setShowLoadTest] = useState(false);
 
-  const reqResObj = currentResponse.url 
-  ? currentResponse : reqResArray.length > 0 
-  ? reqResArray[reqResArray.length - 1] : null;
+  const reqResObj = currentResponse.url
+    ? currentResponse
+    : reqResArray.length > 0
+    ? reqResArray[reqResArray.length - 1]
+    : null;
+
+  const getDisabledReason = () => {
+    if (isTestRunning) {
+      return 'Load test is currently running.';
+    } else if (!reqResObj) {

```

Comment:
Nit, and let's do it in the next PR:

```suggestion
  const getDisabledReason = (): string | null => {
    if (isTestRunning) {
      return 'Load test is currently running.';
    } else if (!reqResObj) {

```